py/rcm_recoverer.py

```diff
@@ -37,43 +36,34 @@ def smart_open(filename, mode="r"):
             fh.close()
 
 
-def generate_rcm(reads_file_name, compressed_file_name, cliques_ids_file_name,
-                 out_file):
-    # Obtain read ids
-    with smart_open(reads_file_name, "r") as fh:
-        ids = [str(record.id) for record in SeqIO.parse(fh, "fasta")]
+def generate_rcm(compressed_rcm_file_name, clique_ids_file_name, out_file):
+    with smart_open(clique_ids_file_name, "r") as fh:
+        compressed_read2clique = [int(s) for s in fh]
 
-    # Obtain compread2clique
-    with smart_open(cliques_ids_file_name, "r") as fh:
-        compread2clique = [int(s) for s in fh]
-
-    with smart_open(compressed_file_name, "r") as fh:
-        idmap = [int(s) for s in fh]
+    with smart_open(compressed_rcm_file_name, "r") as fh:
+        rcm = [line.split("\t") for line in fh]
+    ids, cliques = zip(*[(id, int(clique)) for id, clique in rcm])
```

Comment:
`cliques` is improper name. `clusters` sounds better. Also let me remind about arbitrary ordering in RCM and input reads files. Also cluster labels in RCM are not necessary numeric, thus, use just `.strip()` instead of `int()`.